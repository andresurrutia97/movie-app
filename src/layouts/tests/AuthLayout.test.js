import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import AuthLayout from '../AuthLayout';

describe('<AuthLayout/>', () => {
  const component = (
    <AuthLayout>
      <div>Test</div>
    </AuthLayout>
  );

  it('should render correctly ', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should coantain a <div/> as a children', () => {
    const wrapper = shallow(component);
    expect(wrapper.children().find('div')).toHaveLength(1);
    expect(wrapper.children().find('div').props().children).toBe('Test');
  });
});
