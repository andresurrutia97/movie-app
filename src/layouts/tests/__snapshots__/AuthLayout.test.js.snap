// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthLayout/> should render correctly  1`] = `
<div>
  <div>
    Test
  </div>
</div>
`;
